packages/sources/crypto-api-v2/README.md

```diff
@@ -0,0 +1,250 @@
+|           |           `network`            |     The blockchain network name     |                `mainnet`, `testnet`                |  `mainnet`  |
+
+### Sample Input
+
+```json
+{
+  "id": "1",
+  "data": {
+    "endpoint": "difficulty",
+    "blockchain": "BTC"
+  }
+}
+```
+
+### Sample Output
+
+```json
+{
+    "jobRunID": "1",
+    "result": 14363025673659.96,
+    "statusCode": 200,
+    "data": {
+        "apiVersion": "2.0.0",
+        "requestId": "60ec5f5205bb78619df200ff",
+        "data": {
+            "item": {
+                "hash": "0000000000000000000d96c617747cc9e1eea35c994c3b2f6f67f0942e04cb90",
+                "height": 690715,
+                "previousBlockHash": "00000000000000000008d1fa3519482540510e3604908e4332c0d3d5b1468d64",
+                "timestamp": 1626102399,
+                "transactionsCount": 2298,
+                "blockchainSpecific": {
+                    "difficulty": "14363025673659.96",
+                    "nonce": 1309992080,
+                    "size": 1441069,
+                    "bits": "387160270",
+                    "chainwork": "00000000000000000000000000000000000000001f71bc5ec7bbc6b5fcd4d2d8",
+                    "merkleRoot": "5f72380e87bd119610cc314994a3cb66adcad182ab1432de513169d3d24b13cd",
+                    "strippedSize": 850758,
+                    "version": 545259524,
+                    "versionHex": "20800004",
+                    "weight": 3993343
+                }
+            }
+        },
+        "result": 14363025673659.96
+    }
+}
+```
+
+### Balance
+
+https://developers.cryptoapis.io/technical-documentation/blockchain-data/unified-endpoints/get-address-details
+
+| Required? |      Name       |                                 Description                                 | Options | Defaults to |
+| :-------: | :-------------: | :-------------------------------------------------------------------------: | :-----: | :---------: |
+|           |   `dataPath`    |                   Path where to find the addresses array                    |         |  `result`   |
+|           | `confirmations` |                           Confirmations parameter                           |         |      6      |
+|           |   `addresses`   | Array of addresses to query (this may also be under the `result` parameter) |         |             |
+
+Addresses is an an array of objects that contain the following information:
+
+| Required? |   Name    |                 Description                  |                  Options                  | Defaults to |
+| :-------: | :-------: | :------------------------------------------: | :---------------------------------------: | :---------: |
+|    ✅     | `address` |               Address to query               |                                           |             |
+|           |  `coin`   |              Currency to query               | `btc`. `eth`, `bch`, `ltc`, `etc`, `bch`, `dash`, `doge` |    `btc`    |
```

Comment:
```suggestion
|           |  `coin`   |              Currency to query               | `btc`, `eth`, `bch`, `ltc`, `etc`, `bch`, `dash`, `doge` |    `btc`    |
```

Reply:
What's the difference here between the chain and coin params? Could we maybe update the description to make it a bit clearer?

Reply:
I also copied this directly from the V1 CryptoAPI adapter.  The `coin` parameter tells the adapter which blockchain to query from.  The `chain` parameter tells the adapter whether to query from the mainnet or testnet.  Do you think we should rename the variables instead of just updating the ReadMe?  I had decided against this to keep the adapter's parameters consistent with V1.

Reply:
Got it. Yeah we should keep the parameter names the same, but I think we can clarify the readme here. What you explained here is a bit clearer than the current description in the table.